samps/contests/mini/config/contest.yaml

```diff
@@ -63,7 +68,6 @@ languages:
     runner: 'java'
     runner-args: '{:basename}'
     use-judge-cmd: true
-    judge-exec-cmd: '/usr/bin/wrapper_java {:timelimit} {:basename}'

```

Comment:
This removes the (machine-specific) definition of the judge's execute command line (which seems appropriate).  However, the previous line still says `use-judge-cmd: true`.  Won't that result in an attempt to use an undefined judge's execute command line?

Note that this same comment applies to all the remaining changes (deletions) in this file.